packages/wallet/src/redux/channel-store/reducer.ts

```diff
@@ -68,10 +76,15 @@ export function signAndStore(store: ChannelStore, state: State): SignResult {
   let channel = getChannel(store, channelId);
 
   const signedState = Signatures.signState(state, channel.privateKey);
+
   if (!validTransition(channel, state)) {
     return {isSuccess: false, reason: "TransitionUnsafe"};
   }
 
+  // if (!validAppTransition(channel, signedState.state, bytecode)) {
```

Comment:
Is there a reason we don't call the new `validAppTransition`?

Reply:
My mistake. Fixing in new commit...